config/routes.rb

```diff
@@ -12,5 +11,5 @@
   resources :template_categories
   resources :analytics
   
-  root 'site#welcome'
+  root 'site#auth'
```

Comment:
This should go to 'templates_url', not 'site#auth', mimicking the original link.